### PR TITLE
[Preview] Update design of custom screenshot and empty screenshot state

### PR DESCRIPTION
**Pull request by Heyrows**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/icons/infoOutlined.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/icons/infoOutlined.svg]: Try to use react-icons instead

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { memo, useState, useRef, Fragment } from "react";
 import { Label, Flex, Image, Button, Text, Spinner } from "theme-ui";
 import { acceptedImagesTypes } from "@lib/consts";
+import InfoOutlined from "../icons/infoOutlined.svg";
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js]: `import { MdInfoOutline } from 'react-icons/md'` ?

File: packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ const dataTipText2 = `The repeatable zone is for a group<br/>
   of fields that you want to be able to repeat an<br/>
   indeterminate number of times, like FAQs`;
 
-const Zones = ({ Model, store, variation, showHints }) => {
+const Zones = ({ Model, store, variation }) => {
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx]: shouldn't there be linter errors here for `no any type`?

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx]: ah wait it's jsx not tsx

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { memo, useState, useRef, Fragment } from "react";
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js]: file has wrong name, it should be `jsx` or `tsx` is we migrate to type scirpt

**Comment by MarcMcIntosh** [comment]
Looks like I can add a custom screen shot.
But I do see the `update` box a lot more than I would like.

Also: should we take this opportunity to migrate a few files to typescript and/or fix some linter warnings?

**Review by MarcMcIntosh — CHANGES_REQUESTED** [review]

File: packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx (hunk 3 above)
Comment (Heyrows) [reply packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx]: I tried to change it to `Tsx`, but then I had to change the upper component too and so one until there was too many mistakes and unknown props that i became a mess :x

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js (hunk 4 above)
Comment (Heyrows) [reply packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js]: (following my comment about my try 👆 ), i'll try for that one :)

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js (hunk 2 above)
Comment (Heyrows) [reply packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js]: you even found it thanks 🙏

File: packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js (hunk 4 above)
Comment (Heyrows) [reply packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js]: worked fined, thanks for telling me to try again 👍  😄

File: packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx (hunk 3 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.jsx]: As long as the codes it's a bit cleaner than when it started it's progress :)

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**